### ISSUE TITLE
Sentinel on master server with 2 NICs is flooding other sentinel logs with -dup-sentinel

### ISSUE THREAD

**Issue by joecoolish** [microsoftarchive/redis#339]
I have 4 servers (Virtual hosted in Hyper-V) each with 2 NICs; one NIC is for Windows Network Load-balancing, the other is for network connectivity.

They are:

M1: 192.168.1.44 / 192.168.1.43
M2: 192.168.1.55 / 192.168.1.54
M3: 192.168.1.66 / 192.168.1.65
M4: 192.168.1.77 / 192.168.1.76

Where the first IP is the WNLB IP address, and the second is the regular network IP

M1 is the Master and the other 3 are Slaves.  That all works fine

Now, each server also has a Sentinel.  If I start M2-M4's sentinels, everything is fine.  If I then start M1's sentinel, all of the other 3 sentinel logs start blowing up with:

```
[DATE] * +sentinel sentinel 192.168.1.43:26379 192.168.1.43 26379 @ webredis 192.168.1.43 6379
[DATE] * -dup-sentinel master webredis 192.168.1.43 6379 #duplicate of 192.168.1.44 or {GUID}
[DATE] * +sentinel sentinel 192.168.1.44:26379 192.168.1.44 26379 @ webredis 192.168.1.43 6379
[DATE] * -dup-sentinel master webredis 192.168.1.44 6379 #duplicate of 192.168.1.43 or {GUID}
```

And if I run `SENTINEL sentinels webredis` on any one of the servers, the M1 sentinel will swap between 192.168.1.44 and 192.168.1.43

Why would the Master sentinel go crazy like this?

**Comment by enricogior** [comment]
Hi @joecoolish 
do you need the Redis servers to listen to both IPs?
If not, use the `bind` configuration flag to listen to just one IP.

**Comment by rpannell** [comment]
@joecoolish As @enricogior stated set up the redis servers to be bound to an ip address.  But second the sentinel needs to announce it's ip address.  I had this same issue a couple of weeks ago and once I got announce-ip and announce-port working, it stopped writing that error in the log file.  The code to allow that to work will be in the next release.  

see the documentation for announce-ip here: http://redis.io/topics/sentinel

**Comment by joecoolish** [comment]
@enricogior

Thanks for the reply!  I added the following line to each of my servers:

`bind 127.0.0.1 192.168.1.43`

Changing the last octet to be the appropriate number.  The problem still existed.  Then I replaced the line with:

`bind 192.168.1.43`

Once again changing the last octet to be the appropriate number.  The problem still existed then.
I read somewhere that adding `bind 0.0.0.0` can fix this issue, it did not.

@rpannell 

As for `announce-ip` support, I added the lines into my sentinel.conf and the sentinel wouldn't start, most likely meaning that the command hasn't been implemented yet in the windows port.  Any word on when the next release that supports that command is being released?
